fictadvisor-api/src/v2/utils/documentation/telegramGroup/delete.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiDocumentationParams } from '../decorators';
+import { TelegramGroupResponse } from '@fictadvisor/utils/responses';
+import { DefaultUnauthorizedResponse } from '../defaultResponses';
+
+export const TelegramGroupDocumentationDeleteTgGroup : ApiDocumentationParams = {
+  isAuth: true,
```

Comment:
name of the documentation object must match such format: `{EntityName}Documentation{MethodName}`